tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -1456,3 +1456,164 @@ def test_ios_during_server_pod_restart(self):
+        """
+        LOGGER.info("STARTED: Verify IOs with retries during server pod restart (kubectl delete)")
+        del_bucket = HA_CFG["bg_bucket_ops"]["no_del_buckets"]
+        del_output = Queue()
+        event = threading.Event()  # Event to be used to send intimation of pod restart
+        t_t = int(perf_counter_ns())
+
+        LOGGER.info("Perform IOs with variable object sizes during server pod restart")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.s3_clean.update(users)
+        access_key = list(users.values())[0]["accesskey"]
+        secret_key = list(users.values())[0]["secretkey"]
+        s3_test_obj = S3TestLib(access_key=access_key, secret_key=secret_key,
+                                endpoint_url=S3_CFG["s3_url"])
+        LOGGER.info("Step 1.1: Perform WRITEs on %s buckets for background DELETEs", del_bucket)
+        test_prefix_del = f'test-44854-del-{t_t}'
+        args = {'test_prefix': test_prefix_del, 'test_dir_path': self.test_dir_path,
+                'skipget': True, 'skipdel': True, 'bkts_to_wr': del_bucket, 'output': del_output}
+        self.ha_obj.put_get_delete(event, s3_test_obj, **args)
+        wr_resp = tuple()
+        while len(wr_resp) != 3:
+            wr_resp = del_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        written_bck = s3_test_obj.bucket_list()[1]
+        assert_utils.assert_equal(len(written_bck), del_bucket,
+                                  f"Failed to create {del_bucket} number of buckets."
+                                  f"Created {len(written_bck)} number of buckets")
+        LOGGER.info("Step 1.1: Performed WRITEs on %s buckets for background DELETEs", del_bucket)
+
+        LOGGER.info("Step 1.2: Perform WRITEs with variable object sizes for parallel READs")
+        test_prefix_read = f'test-44854-read-{t_t}'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=test_prefix_read, skipread=True,
+                                                    skipcleanup=True)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 1.2: Performed WRITEs with variable sizes objects for parallel READs.")
+
+        LOGGER.info("Step 2: Start WRITEs, READs and DELETEs with variable object sizes "
+                    "during server pod restart using kubectl delete")
+        LOGGER.info("Step 2.1: Start WRITEs with variable object sizes in background")
+        test_prefix_write = f'test-44854-write-{t_t}'
+        output_wr = Queue()
+        event_set_clr = [False]
+        args = {'s3userinfo': list(users.values())[0], 'log_prefix': test_prefix_write,
+                'nclients': 2, 'nsamples': 10, 'skipread': True, 'skipcleanup': True,
+                'output': output_wr, 'setup_s3bench': False, 'event_set_clr': event_set_clr,
+                'max_retries': 5}
```

Comment:
Is this going to depend on some parameter. Can we add this in HA config as a common value as we are going to use it at many places.